task.php

```diff
@@ -32,7 +32,7 @@
 
 // Are tasks set to run via cron instead & are we accessing this file via the CLI?
 // php task.php [tid]
-if(PHP_SAPI == "cli")
+if (is_cli()) {
```

Comment:
There should be no space between `if` and `(`.

Also, the function isn't needed at all (only slows code down a tiny bit), you can just change the conditional.